README.md

```diff
@@ -3,39 +3,100 @@
 ## Installation
 1. Clone repository into workspace [It doesn't have to be called moveit_simple_ws]
 ```
-mkdir moveit_simple_ws && cd moveit_simple_ws
-mkdir src && cd src
-catkin_init_workspace
+export CATKIN_WS=$HOME/moveit_simple_ws/
+mkdir -p $CATKIN_WS/src && cd $CATKIN_WS/src
 git clone -b kinetic-devel https://github.com/plusone-robotics/moveit_simple.git
 ```
 
-2. Source Dependencies
+2. Get Source Dependencies
 ```
+cd $CATKIN_WS/src
 wstool init . moveit_simple/.travis.rosinstall
-cd ..
 ```
 
-3. Package Dependencies
+3. Install Package Dependencies
 ```
+cd $CATKIN_WS
 rosdep install --from-paths src --ignore-src --rosdistro $ROS_DISTRO -y
 ```
+4. Configure and Build
+```
+cd $CATKIN_WS
+catkin init -w .
+catkin config --extend /opt/ros/$ROS_DISTRO \
+              --blacklist \
+                    motoman_bmda3_support \
```

Comment:
Out of curiosity, do we know where the `motoman_*` dependencies come from?

Reply:
They get downloaded in `motoman-experimental` which is in the rosinstall